### DISCUSSION TITLE
［一點源黑］延續漢字傳統之美，需要您的參與！

### DISCUSSION THREAD

**Discussion by i-font** [ichitenfont/FuturePlans#6] [公告]
傳統漢字印刷體，於雕刻、鉛印、照排等媒介上，以傳承字形承載了漢字數百年工匠的經驗及歷代文字學學者的字理，讓漢字在美學上和構字科學上都力臻完善境界。

如今，志願團體「一點字坊」於數字科技媒體上延續前賢的精華，推出開源字型和形體參考文件，現已經有一定成果。爲進一步推動傳承字形成果，我們正在開展「一點源黑」開源黑體字型計畫，基於思源黑體字型製作修改傳承字形版，需要您的協助！

若您對字型製作、修改有興趣，並希望爲漢字的美學和文字學傳承添磚加瓦，歡迎加入我們的行列！
有意者請在本discussion下留言，感謝！

![一點源黑製作計劃招募中](https://github.com/ichitenfont/.github/assets/8178284/103a0e22-3aad-4360-84c8-a545537daef6)
![一點源黑製作計劃招募中](https://github.com/ichitenfont/.github/assets/8178284/b99670c5-fd17-448f-9ce3-be48672067e1)

**Comment by garfieldLSY** [comment]
您好，各位老師，我是一名理科生，所以可能幫不上甚麽忙，但我願意出一份力，可以嗎？

**Comment by laabaafu** [comment]
你好，我雖不懂字體設計，但願意出力。

**Comment by ghost** [comment]
由於是思源黑體的授權方式，這款字體也是免費商用的。

**Comment by MY1L** [comment]
例图中逗号的字重过细，我想逗号外观也需要修改。
有些问题：
1. 例图为多字重，那么这个是可变字体吗？
2. 如果不可变，那么是否为3次曲线？或者源文件是3次曲线的？
3. 我正在制作一些思源黑没有的搭配思源黑风格汉字（只有常规字重，3次曲线）由于上述2个问题，不清楚你们是否用得着。

**Comment by hiohlan** [comment]
您好，我中文不流利，只能創造一些泰文字型(quadratic curve, FontForge)， 但願意貢献。

**Comment by birchmoonlight** [comment]
我会一点字体制作，也懂一点文字学知识，愿意加入

**Comment by CoolMarvel43** [comment]
- https://github.com/CoolMarvel43/SHS-UFO-Edits/tree/main/Sources/All-Traditional
- https://github.com/CoolMarvel43/SHS-UFO-Edits/tree/main/Sources/Missing-JP-Glyphs
- https://github.com/CoolMarvel43/SHS-UFO-Edits/tree/main/Sources/V1-Revival

我的中文不太熟練，因此我的 GitHub 儲存庫的自述文件是英文的，但您可以查看我對 UFO 格式的思源黑體/宋體的編輯（僅適用於 ExtraLight 和 Heavy 母版）。如果可用的字形不多，我表示歉意，但我會隨著時間的推移添加新的字形來更新我的儲存庫。其中只有一半被認為有用，因為我主要關注 70 年代的中文照排字形。

**2025 年更新**：我更新了鏈接，並添加了幾個新的存儲庫。其中一個儲存庫包含 Adob​​e 從未在主思源黑體字體中包含的所有 JP 風格字形，以優化設計。我還將 v1 字形的修復工作單獨存放到了一個資料夾中。

除了早期經典 (CL) 字形之外，主資料夾中可能還會有新的現代 (MN) 字形，但 CL 字形仍將是預設字形。任何適用於傳承字形的字形現在都將以 ".inherited" 或 ".inherited2"（異體）表示。

My Chinese is not well-versed, therefore this repository’s readme is in English, but you can check out my edits of Source Han Sans/Serif in UFO format (only available in ExtraLight and Heavy masters). I apologise if there are not many glyphs available, but I will update my repository by adding new glyphs over time. Only half of the available glyphs are deemed useful because I am mostly focused on 1970s Chinese phototypesetting orthographies.

**2025 update**: I have updated the links and included a few new repositories. One contains all the JP-style glyphs that Adobe never included in the main Source Han Sans font, to make the design better. I also separated the v1 glyph restoration work to its own folder.

There may be new modern forms alongside Classic forms in the main folder, but Classic forms will remain the default. Any glyphs that are suitable for Inherited Glyphs will now be indicated with ".inherited" or ".inherited2" (variant).

PDF 預覽 / PDF Preview :
- https://github.com/CoolMarvel43/SHS-UFO-Edits/blob/main/Sources/All-Traditional/Sans/Preview/WIPSHDC-All-Traditional-Sans-Regular.pdf
- https://github.com/CoolMarvel43/SHS-UFO-Edits/blob/main/Sources/Missing-JP-Glyphs/Sans/Preview/WIPSHDC-Missing-JP-Sans-Regular.pdf
- https://github.com/CoolMarvel43/SHS-UFO-Edits/blob/main/Sources/V1-Revival/Sans/Preview/WIPSHDC-V1-Revival-Sans-Regular.pdf

**Comment by rwxguo** [comment]
平時比較忙，但如有需要，希望隨緣參與，出一份力。

**Comment by jisbig5** [comment]
你好,我不會字形繪製 只會FontCreator   glyph複製粘貼 希望能出一份力

**Comment by KunYuBoGU** [comment]
我來晚了，請把我帶上吧！！！！

**Comment by CarrotDLaw** [comment]
🙋‍♂️

**Comment by TraditionalChinese** [comment]
**請問有沒有羣之類的可以討論交流呢，**目前覺得這是非常好的，應該讓人們意識到印刷字體應該和手寫體字形上的區別，現在的系統自帶字體像微軟的新細明體之類的，明明是宋體的印式，應該用印刷體的字形才對，可是卻用手寫體的字形弄在宋體的身上，成了一個一倫不類的字體，如果是楷體用手寫體的樣式還還說得過去，因爲楷體本身模仿的就是手寫體。

**Comment by ZN-nini** [comment]
怎么参与

**Comment by catom2000** [comment]
相對測試掌握了Opentype Feature 異書功能，要涵蓋一點明體所有可接受異體字，需要長時間完成，但仍會出力！

**Comment by 123ceq** [comment]
作为一位高中生汉字爱好者我也想出一份力

**Reply by ghost** [reply]
＋1

**Comment by Kriled** [comment]
现在还能参与帮忙吗？我想参与

**Comment by 123ceq** [comment]
能




---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2025年3月8日 13:44 |
| 收件人 | ***@***.***> |
| 抄送人 | ***@***.***>,
***@***.***> |
| 主题 | Re: [ichitenfont/FuturePlans] ［一點源黑］延續漢字傳統之美，需要您的參與！ (Discussion #6) |

现在还能参与帮忙吗？

**Reply by Kriled** [reply]
有参加渠道吗

**Comment by KathrynCG** [comment]
我也想參與

**Comment by Cju-Jyo-Phiung** [comment]
有一点点文字學知識，自己弄過一些FontCreator的小修小改，非常認同製作組的理念，希望能夠參與學習一下。

**Comment by Shawwwwwwwwwwww** [comment]
一名品牌设计师，想要参与到这个字体的创立

**Comment by spew8712** [comment]
一點明體惠我良多，也頗認同檢校表之理據，並有些許fontcreator經驗，希望能幫上忙。

**Comment by Shabibz** [comment]
你們「一點」的人可不可以要點臉，做好自己的字體就行了，為什麼要去干涉別的字體？如果你們的字體做得好，自然會有人去用，沒必要跑到別的字體那裡攻擊有不同意見的人。

**Comment by vatnid** [comment]
作為喜愛字體的語言迷，很認同一點明體和一點源黑的理念，希望能以任何形式參與

**Comment by FQ-7420** [comment]
您好，請問一下進度如何了？我對這個非常感興趣，希望能夠幫上忙

**Reply by birchmoonlight** [reply]
搁置中，因为积极的人手不够

**Reply by birchmoonlight** [reply]
但要补字也可以发字表去做

**Reply by deepsleep-v3** [reply]
可以把未完的源代码等开放出来吗，我对你们非常信赖，但我等不及了
